python/eups/db/ChainFile.py

```diff
@@ -186,31 +194,28 @@ def _read(self, file=None, verbosity=0):
         fd = open(file)
 
         flavor = None
-        lineNo = 0           # line number in input file, for diagnostics
-        for line in fd.readlines():
-            lineNo += 1
-            line = line.strip()
-            line = re.sub(r"#.*$", "", line)
-            if not line:
+        for at, line in enumerate(fd):
+            line = line.lstrip()  # remove any leading whitespace
+            if not line or line.startswith('#'):
```

Comment:
I suspect some ancient hand-written files may have them, but certainly nothing that's been written by Eups in the past ~6-7 years. I don't think there's a huge risk of a regression.